egs2/ami/vad1/run.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+#  Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# You may set 'mic' to:
+#  ihm [individual headset mic- the default which gives best results]
+#  sdm1 [single distant microphone- the current script allows you only to select
+#        the 1st of 8 microphones]
+#  mdm8 [multiple distant microphones-- currently we only support averaging over
+#       the 8 source microphones].
+# ... by calling this script as, for example,
+# ./run.sh --mic sdm1
+# ./run.sh --mic mdm8
+mic=ihm
+
+train_set=${mic}_train
+valid_set=${mic}_dev
+test_sets="${mic}_eval ${mic}_dev"
+
+vad_config=conf/tuning/train_vad_rnn.yaml
+inference_config=conf/tuning/decode_rnn.yaml
+
+./vad.sh \
+    --ngpu 1 \
+    --local_data_opts "--mic ${mic}" \
+    --vad_config "${vad_config}" \
+    --inference_config "${inference_config}" \
+    --train_set "${train_set}" \
+    --valid_set "${valid_set}" \
+    --test_sets "${test_sets}" \
+    --local_data_opts "--stage 2" \
+    --gpu_inference true
```

Comment:
Would be great if you can make them as the default (and let user to change them depends on their needs)